pages/[...error].vue

```diff
@@ -1,5 +1,10 @@
 <script setup lang="ts">
+const { t } = useI18n();
 const { params } = useRoute<"error">();
+toolbarTitleStore().setReactiveToolbarTitle(() => t("nav.unknown"));
```

Comment:
not sure if it's a good idea or not. But maybe we could just use the Toolbar title if no useHead.title is provided.
That would simplify a lot of pages

Reply:
True.. my first thought was that this wouldn't be in line with what I agreed on with Søren, but when I now think it through, it seems also hard to realize without having some glitches here and there 😅 I think we'll have to define it at the initialization of every page, and I think this makes you also aware of that there are two titles which you have to decide how you want them to be.

Reply:
@SimonSimCity Can you have a look at my commit and give feedback on it?